src/app/features/settings/components/settings-form/settings-form.component.html

```diff
@@ -0,0 +1,95 @@
+<form class="w-1/2" [formGroup]="form" (ngSubmit)="submit()">
+  <p class="text-blue-600 font-bold mt-4">General</p>
+  <p class="form-input-label">Avatar</p>
```

Comment:
Labels should be real labels: when you click on the it, corresponding input should react (text input is focused, file input triggers file selection, etc.). So not `p`, it definitely should be `label` tag. If don't understand how to tie it to your custom file input, you probably build it not very well, maybe consider reshaping it